lib/ask/activity_log.ex

```diff
@@ -103,21 +103,28 @@ defmodule Ask.ActivityLog do
   defp typeof(%Questionnaire{}), do: "questionnaire"
   defp typeof(%Folder{}), do: "folder"
 
+  defp remote_ip(nil), do: "0.0.0.0"
+
+  defp remote_ip(%{remote_ip: remote_ip, req_headers: headers}) do
+    case Enum.find(headers, fn {key, _value} -> key == "x-forwarded-for" end) do
```

Comment:
shouldn't we make this condition case insensitive?

Reply:
Plug states that ["request headers are normalized to lowercase"](https://hexdocs.pm/plug/1.12.1/Plug.Conn.html)